src/hardware/openbench-logic-sniffer/protocol.h

```diff
@@ -50,20 +50,26 @@
 /* Trigger config */
 #define TRIGGER_START              (1 << 3)
 
-/* Bitmasks for capture_flags */
-/* 12-13 unused, 14-15 RLE mode (we hardcode mode 0). */
-#define CAPTURE_FLAG_INTERNAL_TEST_MODE (1 << 11)
-#define CAPTURE_FLAG_EXTERNAL_TEST_MODE (1 << 10)
-#define CAPTURE_FLAG_SWAP_CHANNELS      (1 << 9)
-#define CAPTURE_FLAG_RLE                (1 << 8)
-#define CAPTURE_FLAG_SLOPE_FALLING      (1 << 7)
-#define CAPTURE_FLAG_CLOCK_EXTERNAL     (1 << 6)
-#define CAPTURE_FLAG_CHANNELGROUP_4     (1 << 5)
-#define CAPTURE_FLAG_CHANNELGROUP_3     (1 << 4)
-#define CAPTURE_FLAG_CHANNELGROUP_2     (1 << 3)
-#define CAPTURE_FLAG_CHANNELGROUP_1     (1 << 2)
-#define CAPTURE_FLAG_NOISE_FILTER       (1 << 1)
-#define CAPTURE_FLAG_DEMUX              (1 << 0)
+#define DEVICE_FLAG_ADVANCED_TRIGGER (1 << 0)
```

Comment:
This line can go. It will be removed in a later patch without being used

Reply:
Yes, that's a remnant from an earlier development state. Removed. Good catch!